packages/design-system/src/styles/settings/variables/_forms.scss

```diff
@@ -9,10 +9,19 @@ $input-border-radius: $border-radius !default;
 $input-border-color: $color-base !default;
 $input-border-color-inverse: $color-black !default;
 
-$choice-checked-background-color: $color-primary !default;
+// Choice theme variables
+$choice-background-color: $color-background !default;
```

Comment:
There is a variable for `background-color` and `border-color` for every choice state (i.e. default, checked, disabled).  In addition,  there are default and disabled variables for inverse choices.

I also added `$choice-border-radius` which will be useful in the Mgov DS